src/NuGet.Core/NuGet.Common/NuGet.Common.csproj

```diff
@@ -6,6 +6,7 @@
     <Description>Common utilities and interfaces for all NuGet libraries.</Description>
     <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
     <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworks);net45</TargetFrameworks>
+    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' == 'true'">$(TargetFrameworks);netstandard2.0</TargetFrameworks>
```

Comment:
off topic for this PR, but given that common.project.props defines `<IsBuildOnlyXPLATProjects>$(DotNetBuildFromSource)</IsBuildOnlyXPLATProjects>`, should we use `DotNetBuildFromSource` instead?

I'm the team's CI "expert", but it's not clear to me what is source-build specific, and what we do in our own CI differently on Windows vs Mac/Linux.  I think if we removed `IsBuildOnlyXPLATProjects` and used `DotNetBuildFromSource` instead throughout our build scripts, it would be much easier to understand what is what.